src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --color-hover: #00acdc;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.img {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  height: 60px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  display: inline-flex;
+  height: 60px;
+  align-items: center;
+  justify-content: center;
+  line-height: 14.06px;
+
+  &:hover {
+    color: var(--color-hover);
+  }
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
```

Comment:
You've added a `margin-right` to the `.nav__item` class. To be consistent with your vertical margins, you should add only top or only bottom margin, but not both. In your case, you should probably add a `margin-left` instead of `margin-right`, and remove the margin for the first child. This way, you won't have an unnecessary margin on the right of the last item.